tasmota/xsns_13_ina219.ino

```diff
@@ -175,15 +218,11 @@ bool Ina219Read(void)
 
 /*********************************************************************************************\
  * Command Sensor13
- *
- * 0 - Max 32V 2A range
- * 1 - Max 32V 1A range
- * 2 - Max 16V 0.4A range
 \*********************************************************************************************/
 
 bool Ina219CommandSensor(void)
 {
-  if ((XdrvMailbox.payload >= 0) && (XdrvMailbox.payload <= 2)) {
+  if ((XdrvMailbox.payload >= 0) && (XdrvMailbox.payload <= 255)) {
```

Comment:
Is the `255` payload intended for retrieving data (like for `gpio 255` command )? 
If yes, why restart the device?

Reply:
Hi @effelle,
No. In this case 255 is a considered a valid payload which declare a resistor of 25 * 10^5 = 2.5 Mohm. As explained in the description above, shunt resistor value is encoded by a decimal number in the form RRM (from 10 to 255) which means a value of MM * 10^M
This allows a large range of values while avoiding to change `Settings.ina219_mode` from a `uint8_t` to a large integer. 

The restart is here from the original code and I didn't change it because I don't know what would be the impact on Tasmota. On the INA219 driver itself, there is no reason to restart.
May be this is to force storage of the value in the flash ?
Any advice will be appreciated

Reply:
If 255 is a payload to define the value ok, a restart is needed to update the value in flash.